src/components/settingsMenu.js

```diff
@@ -0,0 +1,104 @@
+/* eslint-disable react/jsx-props-no-spreading */
+import React from 'react'
+import PropTypes from 'prop-types'
+import {
+  Box,
+  Tab,
+  Tabs,
+  AppBar,
+  Typography,
+} from '@material-ui/core'
+
+import GurbaniSwitchToggle from './editors/gurbaniToggles'
+import LayoutEditor from './editors/layoutEditor'
+import BackgroundEditor from './editors/backgroundEditor'
+import FontEditor from './editors/fontEditor'
+import VishraamEditor from './editors/vishraamEditor'
+import ExportEditor from './editors/exportEditor'
+
+// WIP: need to organize code
+
+const TabPanel = props => {
+  const {
+    children, value, index, ...other
+  } = props
+
+  return (
+    <div
+      role="tabpanel"
+      hidden={value !== index}
+      id={`scrollable-auto-tabpanel-${index}`}
+      aria-labelledby={`scrollable-auto-tab-${index}`}
+      {...other}
+    >
+      {value === index && (
+        <Box p={2}>
+          <Typography>{children}</Typography>
+        </Box>
+      )}
+    </div>
+  )
+}
+
+TabPanel.propTypes = {
+  children: PropTypes.node,
+  index: PropTypes.any.isRequired,
+  value: PropTypes.any.isRequired,
+}
+
+const tabPanelProps = index => ( {
+  id: `scrollable-auto-tab-${index}`,
+  'aria-controls': `scrollable-auto-tabpanel-${index}`,
+} )
+
+
+const EditorPanel = () => {
+  const [ value, setValue ] = React.useState( 0 )
+
+  const handleChange = ( event, newValue ) => {
```

Comment:
one-line this, and change `event` to eitehr `_` or `_event` to show it is not being used